### PR TITLE
[Feat] README.md: Embed latest ARNEIS YouTube video 

### PR DESCRIPTION
**Pull request by gmacario**
## 📑 Description

Update video URL as a consequence of #246 

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed

### CODE REVIEW

**Comment by gmacario** [comment]
Do not merge until #120 is resolved

**Comment by gmacario** [comment]
Now that the webinar is public let's review and merge this PR

**Comment by gmacario** [comment]
@alv67 and @gteti since this will change the top-level README.md could you please double check before I merge this PR?

**Review by gteti — APPROVED** [review]

**Review by alv67 — APPROVED** [review]

**Pull request merged**
